src/ros_filter.cpp

```diff
@@ -2401,15 +2401,15 @@ namespace RobotLocalization
     // It's unlikely that we'll get a velocity measurement in another frame, but
     // we have to handle the situation.
     tf2::Transform targetFrameTrans;
-    bool silentchecktf;
-    nh_.getParam("/silent_tf_failure",silentchecktf);
+    bool silent_tf_failure;
+    nh_.getParam("/silent_tf_failure",silent_tf_failure);
```

Comment:
The build is reporting failure because it needs to be linted:

http://build.ros.org/job/Mpr__robot_localization__ubuntu_bionic_amd64/25/testReport/junit/roslint/LintCheck/roslint_robot_localization/

You need a space after the comma:

`nh_.getParam("/silent_tf_failure", silent_tf_failure);`